reference/twig_reference.rst

```diff
@@ -324,6 +324,11 @@ Returns the relative URL (without the scheme and host) for the given route.
 If ``relative`` is enabled, it'll create a path relative to the current
 path. More information in :ref:`templating-pages`.
 
+.. note::
+
+    ``parameters`` accept some special parameters. More information in
+    :ref:`special-routing-parameters`.
```

Comment:
As you need to pass all required route parameters with this argument (not only "special" parameters) I would rather link to the routing guide itself (or maybe even better link to http://symfony.com/doc/current/templating.html#linking-to-pages).